platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/binding/api/impl/ValidatorImpl.java

```diff
@@ -101,6 +101,10 @@ public void validateRelayState(String relayState) {
     if (relayState != null && relayState.length() > 80) {
       LOGGER.warn("RelayState has invalid size: {}", relayState.length());
     }
+
+    if (relayState != null && (relayState.contains("<") || relayState.contains(">"))) {
```

Comment:
✏️ Now that we have two null checks here, it may be worth just exiting out if it's null. Though I know some developers prefer not to do that.